packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+        }
+    }
+}
+
+```
+
+## User Profile Rule Object
+
+| Property    | Type                                                                          | Description                                                                           |
+|-------------|-------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| name        | String                                                                        | Human-readable name for the Rule, configurable during creation or updating.           |
+| id          | String                                                                        | Unique identifier for this Rule (read-only).                                          |
+| type        | String                                                                        | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:SignOn`.  |
+| priority    | Integer                                                                       | Used to determine which Rules take. precedence.                                       |
+| conditions  | Array                                                                         | UserType, User, Group and various Device conditions are supported for this rule type. |
+| action      | String                                                                        | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.            |
+| requirement | [User Profile Rule Requirement Object](#user-profile-rule-requirement-object) | Specifies profile attributes to prompt user for.                                      |
+| status      | String                                                                        | `ACTIVE`  or  `INACTIVE`.                                                             |
+| default     | Boolean                                                                       | `true` for the first instance of this Rule, which gets created by default.            |
+
+#### User Profile Rule Requirement Object
+
+| Property                   | Type   | Description                                                                                                                                                                                                                                    |
+|----------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| preRegistrationInlineHooks | String | Optional. The valid ID of an Inline Hook. Sets a Registration Inline Hook to invoke.                                                                                                                                                           |
```

Comment:
This is an array not a string. It may contain only 1 string right now, which is a valid id of an inline hook.

Reply:
Changed to;
| preRegistrationInlineHooks | Array | Optional. An array that can currently contain only one element. The element must be the unique ID of a Registration Inline Hook to invoke.                                                                                                     |